ViewModels/ScopeViewModel.cs

```diff
@@ -499,6 +503,8 @@ async Task<bool> doAcquireAsync()
 
         updateLaserProperties();
 
+        doMatchAsync();
```

Comment:
Hmm. I think matching should be optional (distinct button-press). Most ENLIGHTEN users aren't going to be looking for real-time matching of every spectrum they take. (Not all spectra will even have the laser firing...)

I'd consider restoring the commented-out matchCmd.

Reply:
I anticipate the design for the data collection and processing flow will change substantially over the next few months if not sooner. For now I prefer to keep it like this because it is impressively fast and looks nice